kong/plugins/ai-proxy/handler.lua

```diff
@@ -4,7 +4,7 @@ local _M = {}
 local ai_shared = require("kong.llm.drivers.shared")
 local llm = require("kong.llm")
 local cjson = require("cjson.safe")
-local kong_utils = require("kong.tools.utils")
+local kong_utils = require("kong.tools.gzip")
```

Comment:
The name of this variable is now misleading, it should be called gzip now and find-and replace on this file.
```suggestion
local gzip = require("kong.tools.gzip")
```